server/bin/pbench-sync-satellite.sh

```diff
@@ -3,10 +3,9 @@
 # Simulate what pbench-move-results does for tar balls found on a given
 # satellite (remote) pbench server.
 
-# pbench-move-results copies tarballs to the server in the reception
-# areas (fs-version-001 and -002 versions), and then the respective
-# shims set about moving them into place for pbench-dispatch to move
-# through the various processing stages.
+# pbench-move-results copies tarballs to the server in the reception area, and
+# then pbench-put-shim sets about moving them into place for the various
```

Comment:
I get that "sets about" comes from the original, but should you come back this way I'd prefer to see that annoying and redundant phrase removed. 😆 How 'bout just "moves them into place", for example.